src/hu_stuff.c

```diff
@@ -694,7 +695,18 @@ static void HU_widget_build_title (void)
 
   M_StringConcat(hud_titlestr, s, sizeof(hud_titlestr));
 
-  HUlib_add_string_to_cur_line(&w_title, hud_titlestr);
+  // [crispy] print the map title in white from the first colon onward
+  if (mapcolor_preset == AM_PRESET_CRISPY ||
+      mapcolor_preset == AM_PRESET_ZDOOM)
+  {
+    s = M_StringReplace(hud_titlestr, ":", (const char[]){ ':', '\x1b', '0'+CR_GRAY, '\0' });
```

Comment:
Crispy has this feature? In GZDoom the first part of the title is red.
It's too late to remove the `hud_titlestr` colouring completely, isn't it? 😄

Reply:
> Crispy has this feature? In GZDoom the first part of the title is red. 

Nit: it's unmodified, so e.g. in btsx_e1 it remains in cyan. 

> It's too late to remove the `hud_titlestr` colouring completely, isn't it? 😄

I'm afraid yes. 😉

Reply:
I vote to remove the extra colouring of the map title.

Reply:
You mean the bi-coloring? Okay, I can `#if 0` it out, maybe forks want to keep it.

Reply:
> You mean the bi-coloring? Okay, I can `#if 0` it out, maybe forks want to keep it.

Yes please. I want to use Crispy preset without this.